components/Vote/201907/BudgetChartItem.js

```diff
@@ -85,101 +85,89 @@ const styles = {
   })
 }
 
-class BudgetChartItem extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      collapsed: true
-    }
-
-    this.toggleCollapsed = () => {
-      this.setState(({ collapsed }) => ({
-        collapsed: !collapsed
-      }))
-    }
-  }
+const BudgetChartItem = ({
+  vt,
+  children,
+  category,
+  height,
+  background,
+  color,
+  total,
+  highlight,
+  last
+}) => {
+  const [colorScheme] = useColorContext()
+  const [collapsed, setCollapsed] = useState(true)
+  const toggleCollapsed = () => setCollapsed(!collapsed)
 
-  render() {
-    const {
-      vt,
-      children,
-      category,
-      height,
-      background,
-      color,
-      total,
-      highlight,
-      last
-    } = this.props
-    const { collapsed } = this.state
+  const hasMore = !!children
+  const ExpandIcon = collapsed ? MdExpandMore : MdExpandLess
+  const iconTitle = vt(
+    `vote/201907/budget/icon/${collapsed ? 'more' : 'less'}/title`
+  )
 
-    const hasMore = !!children
-    const ExpandIcon = collapsed ? MdExpandMore : MdExpandLess
-    const iconTitle = vt(
-      `vote/201907/budget/icon/${collapsed ? 'more' : 'less'}/title`
-    )
+  const compact = !!height && height < 35
+  const minHeight = compact ? 25 : 35
 
-    const compact = !!height && height < 35
-    const minHeight = compact ? 25 : 35
-
-    return (
-      <Fragment>
+  return (
+    <Fragment>
+      <div
+        {...styles.wrapper}
+        style={{
+          background,
+          borderBottomWidth: collapsed ? 1 : 0,
+          borderBottomStyle: 'solid'
+        }}
+        {...colorScheme.set('borderBottomColor', 'default')}
+      >
         <div
-          {...styles.wrapper}
+          {...styles.toggle}
+          onClick={hasMore ? toggleCollapsed : undefined}
           style={{
-            background,
-            borderBottom: collapsed ? '1px solid #fff' : undefined
+            color: highlight ? colorScheme.getCSSColor('text') : color,
```

Comment:
`getCSSColor` isn't allowed to be used in a style object, but you should be able to easily apply it to the div:
```js
<div {...colorScheme.set('color', highlight ? 'text' : color)} />
```